src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx

```diff
@@ -177,6 +177,14 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Relic infrastructure agent (`SystemSample`, `StorageSample`, `NetworkSample`, and `ProcessSample`) get aggregated into buckets to improve the query performance over longer periods of time.
+NRQL queries scoped to under 1 hour of time will scan the raw data. For queries scoped over 1 hour, data will be scanned in the following aggregated buckets:
+*  1 minute (scopes of 1 hour to 6 hours)
+*  10 minutes (scopes of 6 hours to 3 days)
+*  1 hour (scopes of 3 days to 14 days)
+*  3 hours (scopes of 14+ days)
+Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024, we've improved the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for correct differentiation between `max()`, `min()`, and `average()` values. 
```

Comment:
```suggestion
Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024, we've improved the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for correct differentiation between `max()`, `min()`, and `average()` values. 
```
```suggestion
Historically, these aggregated event buckets only contained one average value per metric. During October and November 2024, we're gradually migrating the aggregation to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) instead. This allows you to differentiate between `max()`, `min()`, and `average()` values. 
```

Reply:
@nr-ksteinbach What do you think of this revision? Also, how can customers check if their metrics have been migrated?